lib/main.js

```diff
@@ -180,12 +208,11 @@ function main() {
         }
         catch (error) {
             if (!isAzCLISuccess) {
-                core.error("Az CLI Login failed. Please check the credentials. For more information refer https://aka.ms/create-secrets-for-GitHub-workflows");
+                core.setFailed("Az CLI Login failed. Please check the credentials. For more information refer https://aka.ms/create-secrets-for-GitHub-workflows");
```

Comment:
what's the difference between .error and .setFailed

Reply:
execution stops here if `setFailed` otherwise it continues.